src/status_im/contexts/wallet/add_account/create_account/new_keypair/confirm_backup/view.cljs

```diff
@@ -99,10 +100,11 @@
         quiz-index             (reagent/atom 0)
         incorrect-count        (reagent/atom 0)
         show-error?            (reagent/atom false)
+
         {:keys [on-success
                 on-try-again
                 masked-seed-phrase
-                theme shell?]} (rf/sub [:get-screen-params])
+                theme shell?]} (quo.theme/use-screen-params)
         unmasked-seed-phrase   (security/safe-unmask-data masked-seed-phrase)
         random-phrase          (reagent/atom [])]
     (fn []
```

Comment:
Hey @Parveshdhull 
Thanks for the PR! it solves one big problem in the codebase. :100: 

I'd have liked that this PR waited for at least two explicit approvals on this PR before merging, just to make sure at least 2 devs checked all the code, since it contains a lot of changes.

In this piece of code, I don't think just replacing the sub for the hook is enough. This is a form-2 component and the hook call is in the initialization function, it violates the hook rules, we should have refactored this piece of code.

I guess it works because it's not expected to be reactive, but I believe if we decided to replace re-frame for hooks, we should also deal with their limitations/rules